interface/src/app/gene/network/network.js

```diff
@@ -5,6 +5,7 @@
 angular.module('adage.gene.network', [
   'ui.router',
   'ui.bootstrap',
+  'ngResource',
```

Comment:
Aha! Never mind, I found it... just wasn't in the changes for this PR. Puzzled why it wasn't a problem to have it missing before, though!